src/azul/bigquery_reservation.py

```diff
@@ -231,7 +263,7 @@ def deactivate(self) -> None:
                     delete_method(name=resource_name)
                     log.info('Deleted resource %r', resource_str)
         self.refresh()
-        if not self.dry_run and self.is_active is not False:
+        if not self.dry_run and self.is_active:
```

Comment:
I overlooked this change on my first review and I'm confused by it. `is_active` has three possible states, `True` (all resources present), `False` (no resources present), and `None` (some but not all resources present). I think `deactivate` should consider anything other than `False` to constitute a failure. So I'm not sure why this change was made or what it has to do with this PR.

Reply:
Ah, I forgot that it captured three states, and thought this was redundant boolean logic. Thanks for catching that!